index.js

```diff
@@ -34,6 +34,8 @@ function username(cwd, verbose) {
 
   if (path.length && path.charAt(0) === '/') {
     path = path.slice(1);
+  } else if (path.length && path.match(/git@.*\:/)) {
+    path = path.split(':')[1];
```

Comment:
Yeah, my initial implementation was suboptimal...
I just pushed a new improved version trying to follow your suggestions. Let me know if you think this could still be improved 😊